packages/forms/examples/forms/pages/demo/demo.page.html

```diff
@@ -1,3 +1,64 @@
-<h1>Forms demo</h1>
+<h1 class="h3 u-margin-bottom">Forms</h1>
 
-<!-- demo the component here -->
+<h2 class="h4 u-margin-bottom">Auto complete example</h2>
+
+<h3 class="h5 u-margin-bottom">Local search</h3>
+
+<div class="u-margin-bottom">
+	<aui-code-snippet [codeSnippet]="AutoCompleteImportExample"></aui-code-snippet>
+</div>
+
+<div class="u-margin-bottom">
+	<aui-code-snippet [codeSnippet]="AutoCompleteExampleJS1"></aui-code-snippet>
+</div>
+
+<div class="u-margin-bottom">
+	<aui-code-snippet [codeSnippet]="AutoCompleteExampleHTML1"></aui-code-snippet>
+</div>
+
+<div class="u-margin-bottom col-md-6 col-xs-12">
+	<div class="a-input">
+		<div class="a-input__wrapper">
+			<aui-auto-complete
+			id="hero-names"
+			placeholder="Choose your hero…"
+			[(ngModel)]="selectedHero"
+			label="name"
+			value="''"
```

Comment:
if it's not being evaluated by [value], shouldn't this just be ""?

Reply:
True